libs/datamodel/connectors/datamodel-connector/src/error.rs

```diff
@@ -62,4 +62,9 @@ pub enum ErrorKind {
         required_count: usize,
         given_count: usize,
     },
+
+    #[error(
+    "Native types can only be used if the corresponding feature flag is enabled. Please add this field in your datasource block: `previewFeatures = [\"previewFeatures\"]`"
+    )]
+    PreviewFeatureDisabled {},
```

Comment:
Is this error limited only to the native types feature? If so, maybe rename the variant to something like `NativeFlagsPreviewFeatureDisabled` (and it doesn't need `{}` here).